src/tests/end-to-end/common/element-identifiers/common-selectors.ts

```diff
@@ -11,3 +11,9 @@ export const CommonSelectors = {
 export const GuidanceContentSelectors = {
     mainContentContainer: '.content-container',
 };
+
+export const fastPassSelectors = {
+    tabstopNavButtonSelector: 'a[title="Tab stops"]',
+    tabStopToggle: 'button[aria-label="Tab stops"]',
+    tabStopToggleAlternativeSelector: 'button[role="switch"]',
```

Comment:
Didn't find any reference to this new selector on this PR... if we're not using it I guess we shouldn't add it

Reply:
I have some refactor that I need to do for this portion of the PR. Will update all the comments after that. But agreed. There are certain selectors that I added when I was experimenting with selection of the radio button but not using them anymore, so will clean that up!